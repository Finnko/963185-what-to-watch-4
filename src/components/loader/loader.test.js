import React from 'react';
import renderer from 'react-test-renderer';
import Loader from './loader.jsx';

const mock = `small`;

describe(`Loader component render correctly`, () => {
  it(`Should Loader component render correctly`, () => {
    const tree = renderer
      .create(
          <Loader
            size={mock}
          />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
