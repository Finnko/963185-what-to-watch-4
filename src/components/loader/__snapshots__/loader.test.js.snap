// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component render correctly Should Loader component render correctly 1`] = `
<div
  className="loader-wrapper"
>
  <div
    className="loader small"
  />
</div>
`;
